Ensure that slock uses musl's version of explicit_bzero(3), instead
of the compat version provided by slock. The compat version is as-is
insufficient to reliably zero memory correctly.

diff -upr a/config.mk b/config.mk
--- a/config.mk	2022-10-04 19:45:59.000000000 +0200
+++ b/config.mk	2024-03-30 15:32:06.061751182 +0100
@@ -25,8 +25,8 @@ COMPATSRC = explicit_bzero.c
 # On *BSD remove -DHAVE_SHADOW_H from CPPFLAGS
 # On NetBSD add -D_NETBSD_SOURCE to CPPFLAGS
 #CPPFLAGS = -DVERSION=\"${VERSION}\" -D_BSD_SOURCE -D_NETBSD_SOURCE
-# On OpenBSD set COMPATSRC to empty
-#COMPATSRC =
+# On systems supporting explicit_bzero(3), set COMPATSRC to empty
+COMPATSRC =
 
 # compiler and linker
 CC = cc
